core/test/two-steps.test.ts

```diff
@@ -126,3 +128,22 @@ test('works with slow route', async () => {
     route: 'slow'
   })
 })
+
+test('back to first step', async () => {
```

Comment:
The verb `back` for `he/she/it` should have `s` on the end.

We are using RSpec naming system.

```suggestion
test('goes back to first step', async () => {
```